supplements/extra-life/infernal-machine-rebuild.xml

```diff
@@ -0,0 +1,211 @@
+            <set name="rarity">Uncommon</set>
+            <set name="stackable">true</set>
+        </setters>
+    </element>
+
+<!-- Thessaltoxin Antidote -->
+    <element name="Thessaltoxin Antidote" type="Magic Item" source="Infernal Machine Rebuild" id="ID_WOTC_IMR_MAGIC_ITEM_THESSALTOXIN_ANTIDOTE">
+        <description>
+            <p>This potion appears to change color and texture each time you look at it. When you drink the potion, it ends the polymorph effect imposed by thessaltoxin.</p>
+        </description>
+        <setters>
+            <set name="category">Potions</set>
+            <set name="cost" currency="gp">0</set>
+            <set name="weight" lb="0.5">½ lb.</set>
+            <set name="type">Potion</set>
+            <set name="rarity">Uncommon</set>
+            <set name="stackable">true</set>
+        </setters>
+    </element>
+
+
+<!-- The Infernal Machine of Lum the Mad -->
+    <element name="The Infernal Machine of Lum the Mad" type="Magic Item" source="Infernal Machine Rebuild" id="ID_WOTC_IMR_MAGIC_ITEM_THE_INFERNAL_MACHINE_OF_LUM_THE_MAD">
+        <description>
+            <p>This strange device was once thought to have been built by gods long forgotten and to have survived the eons since their passing, for it is incredibly ancient and crafted by means unlike anything known today. However, its true origins derive from a planar craft that crashed in the Barrier Peaks, for the Infernal Machine once functioned as this craft's central command console.</p>
+            <p class="indent">Explorers who discovered the crash site removed the command console and brought it back to civilization, not understanding its true purpose or powers. In later years, the console came into the possession of Baron Lum. By experimenting with its many controls, he was able to achieve astounding and destructive magical effects in the building of his empire, and so the artifact came to be known as the Infernal Machine of Lum the Mad. Yet with Lum's eventual defeat, his great machine was destroyed.</p>
+            <p class="indent">The Infernal Machine is a delicate, intricate, bulky, and heavy device, weighing some 5,500 pounds. It can be operated using a control panel containing sixty levers, forty dials, twenty switches, and a number of jeweled components. These controls generate all kinds of powers and effects, as noted below.</p>
+            <p class="indent"><b><i>Silver Wire.</i></b> The Infernal Machine's great size makes it largely immobile. To make ongoing use of the machine (such as while adventuring), it can be connected to its attuned user by a silver wire, a supply of which can always be produced from the machine's inner workings. This silver wire shares the same general nature as the silvery cord of an astral projection spell, connecting to the body of the user and trailing behind them. When so attached, the wire becomes invisible, astral, and extends to virtually infinite length.</p>
```

Comment:
```suggestion
            <p class="indent"><b><i>Silver Wire.</i></b> The <i>Infernal Machine</i>'s great size makes it largely immobile. To make ongoing use of the machine (such as while adventuring), it can be connected to its attuned user by a silver wire, a supply of which can always be produced from the machine's inner workings. This silver wire shares the same general nature as the silvery cord of an <i>astral projection</i> spell, connecting to the body of the user and trailing behind them. When so attached, the wire becomes invisible, astral, and extends to virtually infinite length.</p>
```